kie-dmn-api/src/main/java/org/kie/dmn/api/core/ast/BusinessKnowledgeModelNode.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.dmn.api.core.ast;
+
+import org.kie.dmn.api.core.DMNType;
+import org.kie.dmn.feel.model.v1_1.BusinessKnowledgeModel;
+
+public interface BusinessKnowledgeModelNode extends DMNNode {
```

Comment:
These AST node interfaces are needed to properly type for events (package org.kie.dmn.api.core.event)